src/components/ItemImage.css

```diff
@@ -0,0 +1,5 @@
+.item__image {
```

Comment:
Maybe you can find another way to scale or fill the background with the pictures, some pictures look a bit off.

Reply:
I will replace some pics. :) I need to find good ones.